File: 1078-remove-outermost-parentheses/remove-outermost-parentheses.py
class Solution(object):
    def removeOuterParentheses(self, s):
        result=[]
        open=0
        for i in range(len(s)):
            if(s[i]=='('):
                open +=1
                if(open>1): result.append(s[i])

            elif (s[i] == ')'):
                open -=1
                if(open>0): result.append(s[i])


        return ''.join(result)            
        

File: 1078-remove-outermost-parentheses/remove-outermost-parentheses.cpp
class Solution {
public:
    string removeOuterParentheses(string s) {
        string result="";
        int open=0;
        for(char ch:s){
            if(ch == '('){
                open +=1;
                if(open>1) result +=ch;
            }
            else if(ch ==')'){
                open -=1;
                if(open>0) result +=ch;
            }
        }
        return result;
    }
};